### PR TITLE
PaymentResponse IDL attribute checks

### PR DESCRIPTION
**Pull request by marcoscaceres**
Applies to PaymentResponse attributes: 

* methodName 
* payerEmail 
* payerName
* payerPhone
* requestId 
* shippingOption
* shippingAddress

### CODE REVIEW

File: payment-request/payment-response/helpers.js [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
 
+test(() => {
+  try {
+    const details = {
+      total: {
+        label: "",
+        amount: { currency: "USD", value: "1.0" },
+      },
+    };
+    const request = new PaymentRequest([{ supportedMethods: "basic-card" }]);
```
Comment (edenchuang) [inline payment-request/payment-response/helpers.js]: I think the "details" should be passed into the constructor.

**Review by edenchuang — CHANGES_REQUESTED** [review]

File: payment-request/payment-response/helpers.js (hunk 1 above)
Comment (marcoscaceres) [reply payment-request/payment-response/helpers.js]: d'oh! good catch.

**Review by edenchuang — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Rebased, fixed merge conflicts

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-09-07 04:57:17
Finished: 2017-09-07 05:07:19


<h2>Failing Jobs</h2>
<ul>

<li>build_css</li>

<li>chrome:unstable</li>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18246)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/272745364)

**Pull request merged**